Rubberduck.Parsing/VBA/BuiltInDeclarationLoader.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NLog;
+using Rubberduck.Parsing.Symbols.DeclarationLoaders;
+using System.Threading;
+
+namespace Rubberduck.Parsing.VBA
+{
+    public class BuiltInDeclarationLoader : IBuiltInDeclarationLoader
+    {
+
+        private readonly IEnumerable<ICustomDeclarationLoader> _customDeclarationLoaders;
+        private RubberduckParserState _state;
+
+        private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
+
+        public BuiltInDeclarationLoader(RubberduckParserState state, IEnumerable<ICustomDeclarationLoader> customDeclarationLoaders)
+        {
+            if (state == null) throw new ArgumentNullException(nameof(state));
```

Comment:
I only did not use braces in the guard clauses since they are generic anyway. However, I will happily convert them to comply with the always use braces convention.
Or should I convert the guard clauses to precondition contracts like `Contract.Requires (state != null)`?